rust_snuba/rust_arroyo/src/processing/strategies/healthcheck.rs

```diff
@@ -49,7 +49,7 @@ impl<TPayload, Next> ProcessingStrategy<TPayload> for HealthCheck<Next>
 where
     Next: ProcessingStrategy<TPayload> + 'static,
 {
-    fn poll(&mut self) -> Result<Option<CommitRequest>, InvalidMessage> {
+    fn poll(&mut self) -> Result<Option<CommitRequest>, PollError> {
```

Comment:
that is the zero-sized type, right? if we do this then we have no way to propagate invalid messages to the dlq anymore